README.md

```diff
@@ -1,13 +1,38 @@
-# Summary
-This package contains type definitions for Corvid by Wix.
-Intended to be used with `corvid-cli`
+<p align="center">
+  <img width="200" src="https://static.wixstatic.com/media/85a3c2_d1356dc7622b48cab7017b24d9fa922c~mv2.png">
+</p>
+<h2 align="center">Corvid Types</h2>
+<p align="center">
+  <b>TypeScript type definitions for <a href="https://www.wix.com/corvid/reference/">Corvid APIs</a></b>
+</p>
+<br>
+
+[![npm version](https://img.shields.io/npm/v/corvid-types.svg?style=flat)](https://www.npmjs.com/package/corvid-types)
+[![Status](http://img.shields.io/travis/com/wix-incubator/corvid-types/master.svg?style=flat)](https://travis-ci.com/wix-incubator/corvid-types)  
+
+> **Important:**
+Features relating to the Local Editor are currently in the stage of alpha release and are still being developed. Future functionality may not be backward compatible. When using these features, you assume the risks associated with the use of such an alpha version.
```

Comment:
Is this copy-paste from the docs? If not, consult Sam

Reply:
yep, just a copy-past